compass_app/app/lib/ui/results/business/search_destination_usecase.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:compass_app/utils/result.dart';
+import 'package:compass_app/data/models/destination.dart';
+import 'package:compass_app/data/repositories/destination/destination_repository.dart';
+
+/// Search Destinations Usecase
```

Comment:
General comment around use cases - it would be really good to call out where they fit in to the architecture at some point, either through a dartdoc here or a larger app overview page. I've used MVVM for a long time now and the teams I've been on have never used use cases.

I find that developers are generally resistant to any additional boilerplate or classes that feel redundant, so I think we have an opportunity here to show the value of each piece of this architecture and demonstrate what the extra boilerplate gets you in return.

Reply:
Thanks! I'll copy your comments into our internal doc in case we want to reintroduce use cases later on